test/test_har_to_mite.py

```diff
@@ -182,42 +183,64 @@ def test_set_headers():
 
 
 def test_cookie_not_in_headers():
-    assert 'Cookie' not in set_request_headers_dict(PAGE_HEADERS)
+    assert "Cookie" not in set_request_headers_dict(PAGE_HEADERS)
 
 
 def test_set_body():
-    assert "name0" in set_request_body('post', PAGE_WITH_BODY)
+    assert "name0" in set_request_body("post", PAGE_WITH_BODY)
 
 
 def test_body_empty_for_get():
-    assert set_request_body('get', PAGE_WITH_BODY) == ""
+    assert set_request_body("get", PAGE_WITH_BODY) == ""
 
 
 def test_parse_urls():
     assert _parse_urls(PAGE_TITLES) == [
-        'https://search.mysite.co.uk/search?q=sport&scope=',
-        'https://www.mysite.co.uk/news',
-        'https://www.mysite.co.uk/news/england1',
+        "https://search.mysite.co.uk/search?q=sport&scope=",
+        "https://www.mysite.co.uk/news",
+        "https://www.mysite.co.uk/news/england1",
     ]
 
 
 def test_extract_and_sort_requests():
-    urls = [page['request']['url'] for page in _extract_and_sort_requests(PAGES_200)]
-    assert urls == ['random_page.url', '302_redirection.url', 'a_page.url']
+    urls = [page["request"]["url"] for page in _extract_and_sort_requests(PAGES_200)]
+    assert urls == ["random_page.url", "302_redirection.url", "a_page.url"]
+
+
+def test_har_convert_to_mite():
+    testfile = os.getcwd() + "/testharconvmite.txt"
+    testfile10secs = os.getcwd() + "/testharconvmite10secs.txt"
+
+    if os.path.exists(testfile):
+        os.remove(testfile)
+    if os.path.exists(testfile10secs):
+        os.remove(testfile10secs)
+
+    har_convert_to_mite("test.har", "testharconvmite.txt", 0)
+    har_convert_to_mite("test.har", "testharconvmite10secs.txt", 10)
+
+    assert os.path.exists(testfile)
+    assert os.path.exists(testfile10secs)
+
+    with open(testfile, "r") as testharoutfile:
+        testharStr = testharoutfile.read()
```

Comment:
One very small comment that it's more pythonnic to use snake_case but I'll send you the tick anyway.

Reply:
My future code will strive for greater pythonicity. At least I've given up Hungarian notation.